Pendulum_iLQR.py

```diff
@@ -6,18 +6,18 @@
 # first define a system and its dynamics
 # we will use an inverted pendulum again
 
-#constants
+# constants
```

Comment:
best practice is having configs in another file and importing them. Not super urgent now, but if you find yourself reusing them elsewhere, then make a separate config file or you risk multiple values for the same config floating around.